specifyweb/frontend/js_src/lib/components/Atoms/DataEntry.tsx

```diff
@@ -65,7 +65,7 @@ export const DataEntry = {
   >(
     'DataEntry.Grid',
     'div',
-    `items-center p-1 -ml-1 gap-2`,
+    `items-center p-1 gap-2`,
```

Comment:
-ml-1 seems to have been added here 3 years ago: https://github.com/specify/specify7/commit/691cc00ffd8f27693b2be76e5d16e733ea96cc0b#diff-2d8f82f3760b387dcbc745011e630a34606900c7031b02082811b5cf41748dd9L208

from what I remember, it was added to prevent focus outline from being cut-off when the button is placed in the first/last column:
![Screenshot 2025-01-02 at 18 40 57](https://github.com/user-attachments/assets/cbece010-1536-4407-a03b-a9acb176eb68)

surprisingly though, it appears that the button was being cut off the way this code was, and is no longer cut off without -ml-1 🙆

Reply:
<img width="145" alt="image" src="https://github.com/user-attachments/assets/39cb7299-6d7d-47b6-af1a-5f574b836f19" />

😂 On `issue-6030`